maker-core/easy-governance-frontend.md

```diff
@@ -0,0 +1,32 @@
+# Easy Governance Frontend
+
+The Easy Governance Frontend (EGF) is a standardized and regulated web application for MKR delegation.
+
+{% hint style="warning" %}
+This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date.
+{% endhint %}
+
+## Usability Goals
+
+The primary goal of the EGF is to provide a standardized and unbiased platform used for delegation. This will include:
+* Viewing [delegates](delegates.md) and their profiles.
+* Viewing [constitutional voter committee](cvc.md) profile pages and members.
+* Viewing voting strategies.
+* Representing the interactions between the above.
+
+A key goal is that MKR Holders lacking context and interest in Maker Governance can still engage effectively.
+
+During pre-game, after the initial launch of the EGF, users will be able to generate DAI against their delegated MKR.
```

Comment:
I assume this comes from this line "The Protocol Delegation System allows those who delegate with it to use MKR as collateral to generate Dai as a Voter Incentive". Should this be part of the EGF doc though?